e2e/grpc-svc-app/grpc_svc_app_test.go

```diff
@@ -0,0 +1,112 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package grpc_svc_app_test
+
+import (
+	"crypto/tls"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	"github.com/aws/copilot-cli/e2e/internal/client"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials"
+)
+
+var (
+	initErr error
+)
+
+var _ = Describe("gRPC Service App", func() {
+	const domainName = "copilot-e2e-tests.ecs.aws.dev"
```

Comment:
Yeah, just to expand on @iamhopaul123's point: @seiichi1101 your code looks good!

It's just that we have limitations with our e2e tests and can run into problem if multiple tests use a domain name. Would you mind updating the manifest file to remove `alias: test.copilot-e2e-tests.ecs.aws.dev` ?

Reply:
No, it's probably won't.
Because protocol version `grpc` of the target group is only working on listener protocol `HTTPS`.
Could you give some advice please? 

> To use the feature on your ALB, choose HTTPS as your listener protocol, gRPC as the protocol version for your target group and register instance or IP as targets for the configured target group. 

https://aws.amazon.com/about-aws/whats-new/2020/10/application-load-balancers-enable-grpc-workloads-end-to-end-http-2-support/

Reply:
Awesome thanks @seiichi1101 ! Then I think we can keep it as is, to be honest I think your integration tests are awesome, and we probably don't need an e2e test for this scenario.